DSharpPlus.Commands/Processors/SlashCommands/RemoteRecordRetentionPolicies/IRemoteRecordRetentionPolicy.cs

```diff
@@ -0,0 +1,16 @@
+using System.Threading.Tasks;
+
+using DSharpPlus.Entities;
+
+namespace DSharpPlus.Commands.Processors.SlashCommands.RemoteRecordRetentionPolicies;
+
+/// <summary>
+/// Provides a means to customize when and which application commands get deleted from your bot.
+/// </summary>
+public interface IRemoteRecordRetentionPolicy
+{
+    /// <summary>
+    /// Returns a value indicating whether the application command should be deleted or not.
+    /// </summary>
+    public Task<bool> CheckDeletionStatusAsync(DiscordApplicationCommand command);
```

Comment:
ValueTask?

Reply:
`Task<bool>` instances are cached, so this is marginally faster for sync code, but it doesn't matter at all anyway - this saves single-digit nanoseconds